### DISCUSSION TITLE
Refunding and updating the inventory accordingly

### DISCUSSION THREAD

**Discussion by edoardolunardi** [craftcms/commerce-stripe#321] [Ideas]
### Description
When refunding an order, I would expect the inventory of the items in the order to be updated accordingly.
Is this a feature of the plugin? If not, it would be really handy 🤝 

### Steps to reproduce

1. Place an order with using the stripe gateway
2. Refund it in the orders section
3. Inventory of purchased items is not updated

### Additional info

- Craft CMS version: 3.7.51
- Stripe for Craft Commerce version: 2.4.3
- PHP version: 8.1.9
- Database driver & version: MySQL 8.0.30

**Comment by edoardolunardi** [comment]
Any update on this?

**Comment by pdaleramirez** [comment]
@edoardolunardi The feature is not yet supported, but you can hook into an event to update the item inventory. Use this event https://craftcms.com/docs/commerce/4.x/extend/events.html#afterrefundtransaction
